### PR TITLE
add mouseup event handler

### PR DESCRIPTION
**Pull request by Schmavery**

### CODE REVIEW

File: packages/reglnative/src/opengl.re [hunk 1]
```diff
@@ -129,6 +130,22 @@ let module Gl : Reglinterface.Gl.t = {
...
+            let x = Sdl.Event.(get e mouse_button_x);
+            let y = Sdl.Event.(get e mouse_button_y);
+            let button =
+              switch Sdl.Event.(get e mouse_button_button) {
+              | 1 => Events.LEFT_BUTTON
+              | 2 => Events.MIDDLE_BUTTON
+              | 3 => Events.RIGHT_BUTTON
+              | _ => failwith "Button not supported"
+              };
+            cb button::button state::Events.DOWN x::x y::y;
```
Comment (bsansouci) [inline packages/reglnative/src/opengl.re]: Can you do `Events.UP` here.

File: packages/reglweb/src/webgl.re [hunk 2]
```diff
@@ -116,6 +117,28 @@ let module Gl : Reglinterface.Gl.t = {
...
+        (
+          fun e => {
+            let button =
+              switch (getButton e) {
+              | 0 => Events.LEFT_BUTTON
+              | 1 => Events.MIDDLE_BUTTON
+              | 2 => Events.RIGHT_BUTTON
+              | _ => assert false
+              };
+            let state = Events.DOWN;
```
Comment (bsansouci) [inline packages/reglweb/src/webgl.re]: `Events.UP`

File: packages/reglnative/src/opengl.re [hunk 3]
```diff
@@ -104,6 +104,7 @@ let module Gl : Reglinterface.Gl.t = {
   let render
       window::(window: Window.t)
       mouseDown::(mouseDown: option mouseDownT)=?
+      mouseUp::(mouseUp: option mouseDownT)=?
```
Comment (bsansouci) [inline packages/reglnative/src/opengl.re]: is `mouseDownT` declared?

Comment (Schmavery) [inline packages/reglnative/src/opengl.re]: It's used in the line above lol

Comment (Schmavery) [inline packages/reglnative/src/opengl.re]: But maybe I should rename it

**Comment by Schmavery** [comment]
Sorry, forgot to push!

File: packages/reglweb/src/webgl.re [hunk 4]
```diff
@@ -91,6 +91,7 @@ let module Gl : Reglinterface.Gl.t = {
   let render
       window::(window: Window.t)
       mouseDown::(mouseDown: option mouseDownT)=?
+      mouseUp::(mouseUp: option mouseDownT)=?
```
Comment (bsansouci) [inline packages/reglweb/src/webgl.re]: Can you also update `mouseDownT` here :p

**Pull request merged**